recipes/howardhinnant_date/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "howardhinnant_date" %}
+{% set version = "2.4.1" %}
+{% set sha256 = "98907d243397483bd7ad889bf6c66746db0d7d2a39cc9aacc041834c40b65b98" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/HowardHinnant/date/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - libcurl
+  run:
+    - libcurl
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/date  # [unix]
+    - test -f ${PREFIX}/include/date/date.h  # [unix]
+    - test -f ${PREFIX}/lib/cmake/date/dateConfig.cmake  # [unix]
```

Comment:
Is there an lib*.so that is produced?

Reply:
Yes, `lib64/libtz.so` on linux.

Reply:
OK great, can you please test that it exists :wink:

Reply:
Done.